From 8ab620024d9253a5b76bad837fdace4ea1d40699 Mon Sep 17 00:00:00 2001
From: "Maciej S. Szmigiero" <mail@maciej.szmigiero.name>
Date: Sun, 28 Apr 2024 10:51:57 +0200
Subject: [PATCH] hwdb: Add a common Logitech M185/M225 mouse variant

Measured/tested on a Logitech N1901 mouse that apparently identifies as
one.

(cherry picked from commit 26b69915fbccd3dc8fff4de602d7ed496be4ce67)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 7d1370ea9b..c29c134942 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -432,6 +432,7 @@ mouse:usb:v046dp4079:name:Logitech G Pro:*
 
 # Logitech Wireless Mouse M185
 mouse:usb:v046dp4008:name:Logitech M185:*
+mouse:usb:v046dp4008:name:Logitech M185/M225:*
 mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:4008:*
  MOUSE_DPI=1000@125
 
